main.go

```diff
@@ -16,8 +16,15 @@ limitations under the License.
 
 package main
 
-import "github.com/kubernetes/kompose/cmd"
+import (
+	"log"
+
+	"github.com/kubernetes/kompose/cmd"
+)
 
 func main() {
-	cmd.Execute()
+	if err := cmd.Execute(); err != nil {
+		log.SetFlags(0)
+		log.Fatal(err)
+	}
```

Comment:
I think the error raised by the cmd package should be handled by main.go.
But some `log.Fatalf` still exist inside the package. I think those logging can be extracted here.